compiler/src/iree/compiler/Codegen/Common/GenericVectorization.cpp

```diff
@@ -269,9 +316,8 @@ getVectorSizes(Operation *op, bool useConfiguredVectorSizes) {
       .Default([&](Operation *) {});
 
   if (vectorSizes) {
-    // This can't identify scalable flags, so pad them with `false`.
-    return std::make_pair(vectorSizes.value(),
-                          SmallVector<bool>(vectorSizes->size(), false));
+    scalableFlags.resize(vectorSizes->size());
```

Comment:
hmm, should this be `.resize(..., false)`, so we can initialize it to false when the size of `scalableFlags` is less than `vectorSizes->size`?